src/main/webapp/WEB-INF/static/js/trialmanager/trial-settings-manager.js

```diff
@@ -203,17 +203,6 @@ window.TrialSettingsManager = (function() {
 
 		// Currently we only have special handling for management details (1) and selection strategy (6)
 		switch (group) {
-			case 1805:
```

Comment:
Does this only affect the the Study Details form?

Reply:
Hi @abatac , 
Compared in master datasetimport and design import performs the same (It shows study_bm_code). 

Thanks,
Gelli